fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorEvaluator.java

```diff
@@ -102,10 +102,12 @@ public ScalarOperator evaluation(CallOperator root) {
         }
 
         // return Null directly iff:
-        // 1. Not UDF
+        // 1. The function is nullable
         // 2. Not in NonNullResultWithNullParamFunctions
         // 3. Has null parameter
-        if (!Catalog.getCurrentCatalog().isNonNullResultWithNullParamFunction(fn.getFunctionName().getFunction())
```

Comment:
Why not add this function to isNonNullResultWithNullParamFunction? The judgment here seems to be designed to be controlled according to this variable

Reply:
OK.